File: guanabara-java/src/com/danillo/algoritmos/lista03/Exercicio29.java
package com.danillo.algoritmos.lista03;

import java.util.Scanner;

public class Exercicio29 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o nome do funcionário: ");
        String nome = entrada.nextLine();
        System.out.println("Salário: ");
        float salario = entrada.nextFloat();
        System.out.println("Tempo de serviço na empresa: ");
        int tempoServicoEmpresa = entrada.nextInt();

        float aumento;
        float novoSalario;

        if(tempoServicoEmpresa < 3){
            aumento = salario * 3/100;
            novoSalario = salario + aumento;
        } else if(tempoServicoEmpresa <= 10){
            aumento = salario * 12.5f/100;
            novoSalario = salario + aumento;
        }else{
            aumento = salario * 20/100;
            novoSalario = salario + aumento;
        }
        System.out.println("Nome: " +nome+ "\nSalário reajustado: R$ " +novoSalario);
    }
}

File: guanabara-python/src/lista03/exercicio29.py
nome = input("Insira o nome do funcionário: ")
salario = float(input("Insira o salário do funcionário: "))
tempo_empresa = int(input("Insira o tempo de serviço do funcionário na empresa: "))

if tempo_empresa <= 3:
    aumento = (salario * 3) / 100
    novo_salario = salario + aumento
    print(f"Salário: {salario}. Salário reajustado (+ 3%): {novo_salario}")
elif tempo_empresa > 3 and tempo_empresa < 10:
    aumento = (salario * 12.5) / 100
    novo_salario = salario + aumento
    print(f"Salário: {salario}. Salário reajustado (+ 12.5%): {novo_salario}")
else:
    aumento = (salario * 20) / 100
    novo_salario = salario + aumento
    print(f"Salário: {salario}. Salário reajustado (+ 20%): {novo_salario}")
